.github/workflows/pr.yaml

```diff
@@ -68,3 +68,16 @@ jobs:
     - run: |
         cd config/registry_image_pruner/image_pruner/
         python3.9 test_prune_images.py
+
+  test_reset_notifications:
+    name: Run reset_notifications tests
```

Comment:
I think we need to run it only if reset notifications script or job definition changed.

Reply:
I copied this logic from previous task `test_prune_images` as I considered this as a best practice in this repo (run test for all PRs), but I can add some filtering to reduce resources.

Reply:
Added job to check code changes, so the tests will run only on code update.